cyral/provider.go

```diff
@@ -84,7 +84,6 @@ func Provider() *schema.Provider {
 
 		ResourcesMap: map[string]*schema.Resource{
 			"cyral_datamap":                     resourceDatamap(),
-			"cyral_identity_map":                resourceIdentityMap(),
```

Comment:
Please keep the old registered resource as `deprecated`. We should just remove it in the next major release. You may use the same code for both, but add a parameter to inform that one of the calls are deprecated. See also next suggestions to understand the whole idea.

```suggestion
			"cyral_identity_map":                resourceRepositoryIdentityMap(true),
```